tests/api/controllers/supervisor_notify_import_remove_mf/supervisor_notify_import_remove_mf.c

```diff
@@ -12,38 +12,71 @@ int main(int argc, char **argv) {
 
   int i = 0;
   WbNodeRef node = wb_supervisor_node_get_from_def("NODE");
-  const int FIELD_COUNT = 8;
-  WbFieldRef mfField[FIELD_COUNT];
-  mfField[0] = wb_supervisor_node_get_field(node, "bool");
-  mfField[1] = wb_supervisor_node_get_field(node, "int");
-  mfField[2] = wb_supervisor_node_get_field(node, "float");
-  mfField[3] = wb_supervisor_node_get_field(node, "vec2");
-  mfField[4] = wb_supervisor_node_get_field(node, "vec3");
-  mfField[5] = wb_supervisor_node_get_field(node, "rot");
-  mfField[6] = wb_supervisor_node_get_field(node, "color");
-  mfField[7] = wb_supervisor_node_get_field(node, "string");
-  int mfFieldCount[FIELD_COUNT];
-  for (i = 0; i < FIELD_COUNT; ++i)
-    mfFieldCount[i] = wb_supervisor_field_get_count(mfField[i]);
+  WbProtoRef proto = wb_supervisor_node_get_proto(node);
+  const char *field_names[] = {"bool", "int", "float", "vec2", "vec3", "rot", "color", "string"};
+  const int FIELD_COUNT = sizeof(field_names) / sizeof(field_names[0]);
+  WbFieldRef mf_field[FIELD_COUNT];
+  WbFieldRef mf_proto_fields[FIELD_COUNT];
+  for (i = 0; i < FIELD_COUNT; ++i) {
+    mf_field[i] = wb_supervisor_node_get_field(node, field_names[i]);
+    mf_proto_fields[i] = wb_supervisor_proto_get_field(proto, field_names[i]);
+  }
+
+  int mf_field_count[FIELD_COUNT];
+  for (i = 0; i < FIELD_COUNT; ++i) {
+    const int count = wb_supervisor_field_get_count(mf_field[i]);
+    mf_field_count[i] = count;
+    const int proto_count = wb_supervisor_field_get_count(mf_proto_fields[i]);
+    ts_assert_int_equal(proto_count, count, "Size of proto field %d not correctly initialized: found %d, expected %d", i, count,
```

Comment:
Presumably, the last 3 params should be "i, proto_count, count"?

Reply:
That is correct; fixed!